vision/vision.proto

```diff
@@ -195,6 +195,19 @@ message Video {
   repeated string tag = 3;
 }
 
+// Describes a recognition descriptor
+message FaceDescriptor {
+
+  // A unique identifier for the descriptor according to UUID v4
+  string uuid = 1; 
+
+  // Algorithm used to extract the descriptor
+  string extractor_algorithm_version = 2;
```

Comment:
Isn't this more like a data format? (closely related to extractor algorithm...). It's the same, just wondering which is better to specify.